test/check-cli

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/python3
+
+import argparse
+import os
+import signal
+import subprocess
+import sys
+
+# import Cockpit's machinery for test VMs and its browser test API
+sys.path.append(os.path.join(os.path.dirname(__file__), "../bots/machine"))
+import testvm
+
+
+def main(image):
+    network = testvm.VirtNetwork(0)
+    machine = testvm.VirtMachine(image, networking=network.host(), memory_mb=2048)
+    machine.start()
+    machine.wait_boot()
+
+    print("ssh -o ControlPath=%s -p %s %s@%s" %
+          (machine.ssh_master, machine.ssh_port, machine.ssh_user, machine.ssh_address))
+
+    # run a command to force starting the SSH master
+    machine.execute('uptime')
+
+    r = subprocess.run(["ssh", "-o", "ControlPath=" + machine.ssh_master,
+                               "-p", machine.ssh_port,
+                               machine.ssh_user + "@" + machine.ssh_address,
```

Comment:
I suggest to assemble the ssh argv prefix above,  and re-use it both here and in the print (but I suppose the print is also just an intermediate debugging thing)

Reply:
Indeed, I had used it for debugging. It might be helpful in the future though. I've moved the common prefix out and printed a message with it. Thanks!